### PR TITLE
docs: Add a quick start example on how to create a template

### PR DESCRIPTION
**Pull request by pawamoy**
Issue #323 

Looks like this:
landing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,9 +26,51 @@ A library and CLI app for rendering project templates.
...
+├── 📁 .git ---------------------------------- # your template is a git repository
+├── 📁 {{project_name}} ---------------------- # a folder with a templated name
+│   └── 📄 {{module_name}}.py.jinja ---------- # a file with a templated name
+└── 📄 {{_copier_conf.answers_file}}.jinja --- # answers are written here
+```
+
+```yaml title="copier.yml"
+# configuration
+_templates_suffix: .jinja
+
```
Comment (Yajo) [inline README.md:45]: This is the new default, so you can remove it. We also get a shorter example.

```suggestion
```

File: README.md [hunk 2]
```diff
@@ -26,9 +26,51 @@ A library and CLI app for rendering project templates.
...
--   Use it in your Python code:
+To create a template:
+
+```bash
+📁 my_copier_template ------------------------ # your template project
+├── 📄 copier.yml ---------------------------- # your template configuration
+├── 📁 .git ---------------------------------- # your template is a git repository
+├── 📁 {{project_name}} ---------------------- # a folder with a templated name
+│   └── 📄 {{module_name}}.py.jinja ---------- # a file with a templated name
+└── 📄 {{_copier_conf.answers_file}}.jinja --- # answers are written here
```
Comment (Yajo) [inline README.md:39]: I think this explains it better. I know it's a nitpick, sorry.

```suggestion
└── 📄 {{_copier_conf.answers_file}}.jinja --- # answers are recorded here
```

Comment (pawamoy) [reply README.md]: No problem, thanks :smile:

**Comment by pawamoy** [comment]
You can squash and merge if you feel this is good to go :slightly_smiling_face:

**Comment by epogrebnyak** [comment]
> /cc @epogrebnyak

Thanks for remembering me in relation to #323 - README makes much more sense now about what copier does and how it works, as far as I can judge.

**Pull request merged**